sequel/lib/criteriaProcessor.js

```diff
@@ -691,6 +691,11 @@ CriteriaProcessor.prototype.prepareCriterion = function prepareCriterion(key, va
       }
 
       break;
+
+    default:
+      var err = new Error('Unknown filtering operator: "' + key + "\". Should be 'startsWith', '>', 'contains' or similar");
+      err.operator = key;
+      throw err;
```

Comment:
Worth noting (while on the topic) that try/catch blocks like this de-optimize the surrounding function.  It's easily fixed by breaking try/catches out into their own tiny functions.  If you think the adapters need to deal with this, let me know and I'll open some placeholder issues on the sql adapters.